omnigibson/object_states/particle_modifier.py

```diff
@@ -111,6 +117,13 @@ def create_projection_visualization(
     # Generate the ComputeGraph nodes to render the projection
     core = Core(lambda val: None, particle_system_name=projection_name)
 
+    # Scale radius and height by the parent scale -- projection always points in the negative-z direction of the
+    # parent frame
+    # We do this AFTER we create the source sphere because the actual projection is scaled in the world frame, whereas
+    # the source sphere is already scaled by its own parent frame
+    projection_radius *= np.mean(parent_scale[:2])
+    projection_height *= parent_scale[2]
```

Comment:
Is it because the projection visualization is not under the link in the USD hierarchy?